docs/src/creating-packages.md

```diff
@@ -418,6 +418,7 @@ This is done by making the following changes (using the example above):
   ```julia
   isdefined(Base, :get_extension) ? (using Contour) : (using ..Contour)
   ```
+- Add `Requires` to `[weakdeps]` in your `Project.toml` file, so that it is only installed on Julia versions that do not support extensions.
```

Comment:
It might be good to mention that it would be listed in both deps and weakdeps, and that 1.9+ knows to make that weak

Reply:
```suggestion
- Add `Requires` to `[weakdeps]` in your `Project.toml` file, so that it is listed in both `[deps]` and `[weakdeps]`.
  Julia 1.9+ knows to not install it as a regular dependency, whereas earlier versions will consider it a dependency.
```

Reply:
bump